content/en/watchdog.md

```diff
@@ -16,62 +16,108 @@ further_reading:
   text: "Collect your traces"
 ---
 
-{{< vimeo 278057125 >}}
+<div class="alert alert-warning">
+Watchdog infrastructure stories are in private beta. <a href="/help">Contact Datadog support</a> to have them enabled for your organization.
+</div>
+
+{{< img src="watchdog/watchdog_page.png" alt="Watchdog page" responsive="true" >}}
 
 ## Overview
 
-Watchdog is an algorithmic feature for APM that automatically detects potential application and infrastructure issues. Watchdog observes trends and patterns in application metrics—like error rate, request rate, and latency—and unexpected behavior. Watchdog evaluates all services and resources without the need to configure a monitor for each service.
+Watchdog is an algorithmic feature for APM performances and infrastructure integration metrics that automatically detects potential application and infrastructure issues. Watchdog observes trends and patterns in:
```

Comment:
Same comment as above: `I would say infrastructure metrics instead of infrastructure integration metrics, since Watchdog Infra, does not only rely on integration metrics.`